src/cli/build.ts

```diff
@@ -180,9 +176,9 @@ export default function (program: RootCmd) {
 
       let zarf = "";
       if (opts.zarf === "chart") {
-        zarf = assets.zarfYamlChart(chartPath);
+        zarf = await assets.zarfYamlChart(chartPath);
       } else {
-        zarf = assets.zarfYaml(yamlFile);
+        zarf = await assets.zarfYaml(yamlFile);
```

Comment:
```suggestion
        zarf = await assets.zarfYaml(yamlFile);
```

Why is this async?

Reply:
The return value is a Promise<string> so the await is needed or you get:

`Type 'Promise<string>' is not assignable to type 'string'.ts(2322)`

Reply:
Fixed these by making the functions in assetsDeployer.ts no be async.